code/modules/materials/Mat_MaterialProcs.dm

```diff
@@ -358,7 +358,7 @@ triggerOnEntered(var/atom/owner, var/atom/entering)
 
 	execute(var/location) //exp and temp both have the location as first argument so i can use this for both.
 		var/turf/T = get_turf(location)
-		if(T.density)
+		if(!T || T.density)
```

Comment:
Should this execute if there's just no turf under it?  (Can that happen?)

At that point the density would be `0`, though, right?

If that's the goal, perhaps the code would be

```
if (T && T.density)
  return
```